utils/zsh/Makefile

```diff
@@ -42,13 +42,27 @@ endef
 define Build/Configure
 	$(call Build/Configure/Default, \
 		--bindir=/bin \
```

Comment:
Is there a need to override the defaults?
https://github.com/lede-project/source/blob/master/include/package-defaults.mk#L79